README.rst

```diff
@@ -1,12 +1,43 @@
 Orange Network
 ==============
 
-Orange Network is an add-on for Orange_ data mining software package. It
-provides network visualization and analysis tools.
+Orange3-Network is an add-on for Orange_ data mining software package. It
+provides network visualization and network analysis tools.
 
 .. _Orange: http://orange.biolab.si/
 
 Documentation is found at:
 
-http://orange-network.readthedocs.org/
+https://readthedocs.org/projects/orange3-network/
 
+Installation
+------------
+
+Install from Orange add-on installer through Options - Add-ons.
+
+To install the add-on with pip use
+
+    pip install orange3-network
+
+To install the add-on from source, run
+
+    python setup.py install
+
+To register this add-on with Orange, but keep the code in the development directory (do not copy it to 
+Python's site-packages directory), run
+
+    python setup.py develop
+
+Windows users have to compile the code. Run
+
+	python setup.py build_ext -i --compiler=msvc install
```

Comment:
I've got no idea why this shows such an indent. Looks fine in my editor. :confused: